webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/RowActions.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Td, ActionsColumn } from '@patternfly/react-table';
+import { APIActions } from '../../../../redux/API';
+import { sprintf, translate as __ } from '../../../../common/I18n';
+import { HOST_PARAM } from './ParametersConstants';
+import { openConfirmModal } from '../../../ConfirmModal';
+import { updateHost } from '../../ActionsBar/actions';
+
+export const RowActions = ({ hostId, param }) => {
+  const dispatch = useDispatch();
+  const onDelete = () =>
+    dispatch(
+      APIActions.delete({
+        url: `/api/hosts/${hostId}/parameters/${param.id}`,
+        key: `${param.id}-DELETE`,
+        successToast: () =>
+          sprintf(__('Parameter %s has been deleted successfully'), param.name),
+
+        errorToast: ({ response }) =>
+          // eslint-disable-next-line camelcase
+          response?.data?.error?.full_messages ||
+          response?.data?.error?.message,
+        handleSuccess: () => {
+          dispatch(updateHost(hostId));
+        },
+      })
+    );
+  const rowActions = [
+    param.associated_type === HOST_PARAM && {
+      title: __('Delete'),
+      onClick: () => {
+        dispatch(
+          openConfirmModal({
+            title: `${__('Delete ')} ${param.name}`,
+            message: __(
+              'This will change the delete the parameter, are you sure?'
+            ),
+            isWarning: true,
+            onConfirm: () => {
+              onDelete();
+            },
+          })
+        );
+      },
+    },
+  ].filter(a => a);
+  return (
+    <Td isActionCell className="parameters-actions">
+      {!!rowActions.length && <ActionsColumn items={rowActions} />}
```

Comment:
The convention for Katello's tabs has been to always show the kebab, and then enable or disable the items inside it. Probably better if we follow that here.

Reply:
@MariSvirik :eyes:

Reply:
Kebab - in those items that don't have a kebab. Are they ever going to have available actions there? If the answer is never, there is no need to add kebab, if yes e.g. they need to enable smth let's show it with a tooltip saying why it's disabled. @jeremylenz

Reply:
The items that dont have kebab are items that can never have an action. (the only action this pr has is "delete param")